labs/lab-2-javascript-game/elicecilgame.js

```diff
@@ -0,0 +1,102 @@
+class Player {
+    constructor(name, health, healsRemaining, wins) {
+        this.name = name;
+        this.health = health;
+        this.healsRemaining = healsRemaining;
+        this.wins = wins;
+        this.maxHealth = health;
+    }
+
+    generateAttackDamage() {
+        var damage = Math.round(Math.random() * 3);      
+        return damage;
+    }
+
+    hitPlayer(damage) {
+        this.health = this.health - damage;
+    }
+
+    heal() {
+        var heal = Math.round(Math.random() * 10);  
+        return heal;   
+    }
+
+    healPlayer(heal) {
+        this.health = this.health + heal;
+        this.healsRemaining = this.healsRemaining - 1;   
+    }    
+}
+
+
+function startGame() {
+    var attack = true;
+    var start = confirm("Would you like to fight the Champion?");
+    var userName = prompt("Tell us your name."); 
```

Comment:
Use an input on the same screen as the Start button! Much less ugh than a prompt.